### ISSUE TITLE
Grid layout sizing issue

### ISSUE THREAD

**Issue by jmjones715** [jmjones715/grid-layout#1]
I cant seem to adjust the size of my grid to match the 320px screen shot.

@hellachella

**Comment by hellachella** [comment]
1.  You need to rename your images using the naming convention.
2. Think about this assignment as having 2 grids. One containing the header and h1/text, one containing the cards with the `<dl>`.  
3. After that, go through each screen-shot, starting with the smallest (xs) and size the units according to what you see on each.

**Comment by jmjones715** [comment]
Ok so name convention as in using dashed and lowercase only right?

**Comment by hellachella** [comment]
Yes exactly. You can also shorten them so they make more sense and are easier to use.

**Comment by jmjones715** [comment]
Ok,

Also, how can I get the images to be smaller and centered and the text to be centred under the images?

I tried using grid-middle but nothing seems to be happening

**Comment by hellachella** [comment]
Try putting them into a figure:

<img width="481" alt="screen shot 2018-01-31 at 4 18 28 pm" src="https://user-images.githubusercontent.com/10465921/35648031-686e9092-06a2-11e8-989f-d5ae6f769f8a.png">

A figure has auto sizing which should match what you see in the screenshots.

**Comment by jmjones715** [comment]
Its a little bit better i guess, but I just realized that the body with is only about half the size of the of the page? Is that an html problem?

Sorry for all the questions
<img width="323" alt="screen shot 2018-01-31 at 4 33 19 pm" src="https://user-images.githubusercontent.com/31852589/35648742-b8c13b10-06a4-11e8-86df-bec79441e981.png">

**Comment by jmjones715** [comment]
Alright I just figured out what my issue was.

It was a pretty stupid mistake. In the css i didn't put the period before the img-flex class so nothing was being called.

Thanks for all of your help

**Comment by hellachella** [comment]
Np. Glad you figured it out.
